packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -324,7 +324,7 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
     }
 
     return (
-      menuIconProps ?
```

Comment:
So menuIconProps is always truthy at this point (it gets assigned if it was undefined and its type is IIconProps | undefined, so it cannot be null when strict null checks is enabled). This means that menuIconProps truthiness check was reduntant and it always returns an Icon.